cmd/app.go

```diff
@@ -160,3 +238,53 @@ HTTP methods, query parameters, and authentication mechanisms.`,
 
 	return swaggerCmd
 }
+
+func validateDetectResourceType(resourceType string) (*webscan.DetectResourceType, error) {
+	resourceTypeEnum, err := webscan.NewDetectResourceTypeFromString(strings.ToUpper(resourceType))
+	if err != nil {
+		return nil, err
+	}
+	return &resourceTypeEnum, nil
+}
+
+func validateDetectResourseModuleSelection(resourceType webscan.DetectResourceType, modules []string) ([]*webscan.DetectResourceModule, error) {
+	moduleEnums := []*webscan.DetectResourceModule{}
+	if len(modules) == 0 {
+		return nil, nil
```

Comment:
Should we expect/require at least one module?

Reply:
No - by default all modules are run for whichever resourse type you set